tests/fib/test_fib.py

```diff
@@ -499,7 +502,48 @@ def ipver(request):
     return request.param
 
 
-def test_hash(fib_info_files, setup_vlan, hash_keys, ptfhost, ipver, set_mux_same_side,
+@pytest.fixture
+def add_default_route_to_dut(config_facts, duthosts, tbinfo):
+    """
+    Add a default route to the device for storage backend testbed.
+    This is to ensure the IO packets could be successfully directed.
+    """
+    if "backend" in tbinfo["topo"]["name"]:
+        logging.info("Add default route on the DUT.")
+        try:
+            for duthost in duthosts:
+                cfg_facts = config_facts[duthost.hostname]
+                for asic_index, asic_cfg_facts in enumerate(cfg_facts):
+                    asic = duthost.asic_instance(asic_index)
+                    bgp_neighbors = asic_cfg_facts["BGP_NEIGHBOR"]
+                    ipv4_cmd_parts = ["ip route add default"]
```

Comment:
Are we adding default route for backend asics as well in multi-asic? Backend asics are skipped in the remove path in L538

Reply:
Yes, I followed the logic here:
https://github.com/Azure/sonic-mgmt/blob/6af8a5c871a2ed4791ef9a72b3903a1b26cc08d8/tests/fib/test_fib.py#L42-L51